src/goto-analyzer/goto_analyzer_parse_options.cpp

```diff
@@ -373,7 +373,8 @@ bool goto_analyzer_parse_optionst::process_goto_program(
     remove_function_pointers(
       get_message_handler(), goto_model, cmdline.isset("pointer-check"));
     // Java virtual functions -> explicit dispatch tables:
-    remove_virtual_functions(goto_model);
+    std::vector<irep_idt> virtual_functions;
```

Comment:
As opposed to adding unnecessary vectors that are discarded, consider overloading ` remove_virtual_functions` to create its own empty vector.